network-api/networkapi/wagtailpages/forms.py

```diff
@@ -26,14 +27,16 @@ def clean_parent(self):
         return parent
 
 
-class BlogPageForm(WagtailAdminPageForm):
+class BuyersGuideArticlePageForm(WagtailAdminPageForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        require_share_fields(self)
+
```

Comment:
What do you think about moving this closer to the rest of the buyer's guide classes? Maybe create a `buyersguide/forms.py` module?

Reply:
Hey Tibor, this makes sense to me. I have created a new `buyersguide/forms.py` file, and have moved both `BuyersGuideArticlePageForm`, as well as `BuyersGuideProductCategoryForm`, since it is BuyersGuide related as well